packages/hoppscotch-common/src/components/importexport/ImportExportSteps/MyCollectionImport.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="select-wrapper">
+    <select
+      v-model="mySelectedCollectionID"
+      autocomplete="off"
+      class="select"
+      autofocus
+    >
+      <option :key="undefined" :value="undefined" disabled selected>
+        {{ t("collection.select") }}
+      </option>
+      <option
+        v-for="(collection, collectionIndex) in myCollections"
+        :key="`collection-${collectionIndex}`"
+        :value="collectionIndex"
+        class="bg-primary"
+      >
+        {{ collection.name }}
+      </option>
+    </select>
+  </div>
+
+  <p class="my-4">
+    <HoppButtonPrimary
+      class="w-full"
+      :label="t('import.title')"
+      :disabled="!hasSelectedCollectionID"
```

Comment:
```suggestion
      :label="t('import.title')"
      :disabled="!hasSelectedCollectionID"
      class="w-full"
```

Let's keep `class` below other props, [Ref](https://eslint.vuejs.org/rules/attributes-order.html).

Reply:
according to the rule, we should keep the globals (category `class` belongs to) above @click only. which its already in. the order of the bound variables & class does not seem to matter, because they're in the same priortity.

Reply:
Yup, it's just a convention that I've been following.

> but in this case doesnt it make it more pleasent to read, if the class is not sandwiched between vue attributes

In the suggested approach, it appears below all component props. Hence, there's a clear distinction. But yeah, it's just a matter of preference.